man/machinery-inspect-docker.1.md

```diff
@@ -0,0 +1,109 @@
+
+## inspect-container — Inspect Container
+
+### SYNOPSIS
+
+`machinery inspect-container` [OPTIONS] IMAGENAME
+
+`machinery inspect-container` [OPTIONS] IMAGEID
+
+`machinery` help inspect-container
+
+
+### DESCRIPTION
+
+The `inspect-container` command analysis a container image by creating, starting and inspecting a container from the provided image
```

Comment:
`analysis`doesn't make sense in this context. Do you mean`analyzes`?

How about:
```
The `inspect-container` command inspects a container image. It creates and starts the container from the provided image before inspection...  
```